src/app/BufferedReadCallback.h

```diff
@@ -131,6 +132,7 @@ class BufferedReadCallback : public ReadClient::Callback
     ConcreteDataAttributePath mBufferedPath;
     std::vector<System::PacketBufferHandle> mBufferedList;
     Callback & mCallback;
+    CHIP_ERROR mLastError = CHIP_NO_ERROR;
```

Comment:
This should probably be called mDataBufferingError?

Reply:
done, thanks